.circleci/config.yml

```diff
@@ -179,6 +179,7 @@ jobs:
   build:
     docker:
       - image: cimg/node:16.16.0
+    resource_class: xlarge
     steps:
```

Comment:
These changes are unrelated to your PR

Reply:
@hirad-deriv the feature branch is having conflicts with master branch, and therefore couldnt merge anything to it. thats why i created this branch to resolve conflict and merge it to feature branch.
Probably these unrelated changes happened in master branch after the conflict.

Reply:
@shaheer-deriv I think we can merge this PR to feature branch to resolve conflicts and review the main PR. It is hard to track your changes in this PR.

Reply:
@hirad-deriv if required, we can merge this PR, and review the feature branch PR https://github.com/binary-com/deriv-app/pull/8093 
so that we don't see unrelated changes in the PR.

Reply:
@amina-deriv sure.. will merge this and lets review the feature branch PR.

Reply:
Feature branch PR: https://github.com/binary-com/deriv-app/pull/8093